docs/_includes/attr-element.adoc

```diff
@@ -20,13 +20,16 @@ Attribute lists:
 
 ==== Positional attribute
 // tag::pos[]
-in an attribute list
+Positional attributes are un-named values at the start of the attribute list.
+The attribute that they are assigned to depends on the type of the element:
 
-not named
+* `icon:` (1) size
+* `image:` and `image::` (1) alt text, (2) width, (3) height
+* Delimited blocks: (1) block name
+* Other inline quoted text: (1) `role`
+* Other blocks and macros: (1) `style`
```

Comment:
Technically block name and style are the same thing. The way I would write this is:

```
Delimited blocks: (1) block name (aka style)
```

Then just leave off the "other block and macros" since there is no universal positional attribute for those.